### PR TITLE
Simplify CoreUpdateMonitor/message.properties: changelog.url → jenkins-version.properties

### PR DESCRIPTION
**Pull request by jglick**
Amends #1812 to avoid messing around with localized strings. Works better from `mvn -pl war jetty:run` (otherwise you see only **NewVersionAvailable**) though I am not sure if that actually regressed in #4351 or if it has long been this way: https://github.com/stapler/stapler/blob/7c126212afd970431bc18647189cd4f682471880/jelly/src/main/java/org/kohsuke/stapler/jelly/CustomJellyContext.java#L176 assumes that `xxx.jelly` and `xxx.properties` are collocated, which they would not be if the former was loaded from `core/src/main/resources/` while the latter was only available in `core/target/classes/`.

### CODE REVIEW

File: core/src/main/resources/hudson/model/UpdateCenter/CoreUpdateMonitor/message_bg.properties [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
+UpgradeProgress=<a href="{1}">\u0418\u0437\u043f\u044a\u043b\u043d\u044f\u0432\u0430 \u0441\u0435</a> \u043e\u0431\u043d\u043e\u0432\u044f\u0432\u0430\u043d\u0435 \u043a\u044a\u043c Jenkins {0}.
 Retry=\
  \u041d\u043e\u0432 \u043e\u043f\u0438\u0442
 UpgradeComplete=\
- \u041e\u0431\u043d\u043e\u0432\u044f\u0432\u0430\u043d\u0435\u0442\u043e \u0435 \u0443\u0441\u043f\u0435\u0448\u043d\u043e
-UpgradeFailed=\
- \u041e\u0431\u043d\u043e\u0432\u044f\u0432\u0430\u043d\u0435\u0442\u043e \u0435 \u043d\u0435\u0443\u0441\u043f\u0435\u0448\u043d\u043e
+ \u041e\u0431\u043d\u043e\u0432\u044f\u0432\u0430\u043d\u0435\u0442\u043e \u043a\u044a\u043c \u043d\u043e\u0432\u0430\u0442\u0430 \u0432\u0435\u0440\u0441\u0438\u044f Jenkins {0} \u043f\u0440\u0438\u043a\u043b\u044e\u0447\u0438. \u0422\u0440\u044f\u0431\u0432\u0430 \u0434\u0430 <a\
+ href="{1}/safeRestart">\u0440\u0435\u0441\u0442\u0430\u0440\u0442\u0438\u0440\u0430\u0442\u0435</a>.
+UpgradeFailed=\u041d\u0435\u0443\u0441\u043f\u0435\u0448\u043d\u043e \u043e\u0431\u043d\u043e\u0432\u044f\u0432\u0430\u043d\u0435 \u043a\u044a\u043c Jenkins {0}: <a href="{2}">{1}</a>.
```
Comment (jglick) [inline core/src/main/resources/hudson/model/UpdateCenter/CoreUpdateMonitor/message_bg.properties]: Versions in `src/filter/resources/` seemed better where a key was defined in both places.

**Review by res0nance — APPROVED** [review]

File: core/src/filter/resources/hudson/model/hudson-version.properties [hunk 2]
```diff
@@ -1 +1,2 @@
+# TODO unused?
```
Comment (oleg-nenashev) [inline core/src/filter/resources/hudson/model/hudson-version.properties]: Last time I checked that, there was some tooling using it. I do not remember what exactly, and it does not matter much at the moment

**Review by oleg-nenashev — APPROVED** [review]
Looks good to me, thanks for the cleanup! Later it could be also exposed through API if needed

**Comment by oleg-nenashev** [comment]
I plan to merge it tomorrow if no negative feedback

**Review by halkeye — APPROVED** [review]

**Comment by daniel-beck** [comment]
Comments in [JENKINS-54596](https://issues.jenkins-ci.org/browse/JENKINS-54596) indicate this breaks other locales.

**Comment by jglick** [comment]
⇒ #4440

**Comment by oleg-nenashev** [comment]
Asking KK about out-of-order release options

image

**Pull request merged**